deploy/crds/kieapp.crd.yaml

```diff
@@ -1871,6 +1871,41 @@ spec:
                                 to generated options in JAVA_OPTS. e.g. '-Dsome.property=foo'
                               type: string
                           type: object
+                        kafkaExt:
+                          description: KafkaExt kafka configuration to be used by
+                            the KieApp
+                          properties:
+                            acks:
+                              description: The number of acknowledgments the producer
+                                requires the leader to have received before considering
+                                a request complete.
+                              type: string
+                            autocreateTopics:
+                              description: Allow automatic topic creation.
+                              type: string
+                            bootstrapServers:
+                              description: A comma separated list of bootstrapServers
+                              type: string
+                            clientID:
+                              description: Identifier to pass to the server when making
+                                requests
+                              type: string
+                            extDisabled:
+                              description: Disable the Kafka Extension
+                              type: string
+                            groupID:
+                              description: Group identifier the group this consumer
+                                belongs
+                              type: string
+                            maxBlockMs:
+                              description: Number of milliseconds that indicates how
+                                long publish method will block
+                              type: string
```

Comment:
as this is number, an integer would be better type for this parameter

Reply:
ms maybe make more sense to use int64, right?

Reply:
converted to int, if they need huge amount of time to block we'll increase